moleculemaker/mmli-backend
# Return SVGs generated by molconvert instead of RDkit

### Problem:
1. SVGs generated by RDKit do not match the figures in the PDF, we need to use output from molconvert instead.
2. Export CDXML request for single page does returns internal server error.

### Approach:
1. Made changes to the background task to read SVG files from result based on chemscraper extractPDF changes that now return SVG files as well. (Based on - [PR](https://gitlab.com/dprl/graphics-extraction/-/merge_requests/49))
2. Return svg read from the results folder.
3. Export CDXML for single page failed due to upstream change to the single page cdxml file name in the results. Fixed by changing file name used in endpoint.

### How to test:
1. Changes deployed to [staging](https://chemscraper.frontend.staging.mmli1.ncsa.illinois.edu/results/example_PDF)
2. Run a new chemscraper job and observe results table to match SVGs generated to the figures in the PDF.
3. Export `CDXML` results for a single page - choose any page.
